tar/chunks/chunks_stats_internal.c

```diff
@@ -73,17 +73,32 @@ chunks_stats_printheader(FILE * stream, int csv, int print_nulls)
 {
 
 	if (csv) {
+		if (fprintf(stream, "%s", "Archive name") < 0) {
+			warnp("fprintf");
+			goto err0;
+		}
+		if (fprintf(stream, ",") < 0)
+			goto err0;
```

Comment:
Normally we'd have
```
...) {
        warnp("fprintf");
        goto err0;
}
```

here, but we don't need the `warnp` when we switch to `print_separator()` in the following commit.

I normally try to have a "perfect" refactoring before changing code, but in this specific case I think it's worth skipping the warning in order to reduce the number of changed lines in the diff.